js/modules/scoreTableInitialization.js

```diff
@@ -114,7 +106,7 @@ function createResultTable() {
     const headRow = document.createElement('tr');
   
     let existingScoreTable = readFromStorage('currentScoreTable');
-    let headers = ['#', existingScoreTable.firstPlayer, existingScoreTable.secondPlayer, 'Player#3'];
+    let headers = ['#', existingScoreTable.firstPlayer, existingScoreTable.secondPlayer];
```

Comment:
what is `existingScoreTable.firstPlayer`? is it an object of the class Player or just a String with player's name?

Reply:
existingScoreTable.firstPlayer  -  Player 1 - it's part of currentScoreTable obj what was made on the base of Player class and info and get name as from it

Reply:
I understand that. These variable `headers` show data in the HTML table.
So why do you need a player object here, not just a string with the player name?